RestaurantsNearMe/Data/YelpRequest/YelpRequestProvider.swift

```diff
@@ -13,21 +13,24 @@ import Networking
 class YelpRequestProvider {
   let activeRequest = CurrentValueSubject<YelpRequest?, Never>(nil)
   private var cancellables = Set<AnyCancellable>()
-  
   init(
     locationProvider: some LocationProviding,
     cloudKitServiceProvider: some CloudKitServiceProviding,
+    offsetProvider: some OffsetProviding,
     baseURLPath: StaticString
   ) {
     locationProvider.currentLocation
-      .map { self.translateToQueryDict(from: $0) }
-      .combineLatest(cloudKitServiceProvider.fetchedAPIKey.map { self.translateToHeader(from: $0.value)})
+      .combineLatest(
+        cloudKitServiceProvider.fetchedAPIKey.map { self.translateToHeader(from: $0.value)},
+        offsetProvider.currentPageOffset
+      )
       .receive(on: RunLoop.main)
-      .sink { locationParameters, authHeader in
+      .sink { [unowned self] location, authHeader, offset in
```

Comment:
still getting into the mindset of using `unowned` more thoughtfully.  In this case the lifetime of the provider that is likely longer than the individual components but it feels somewhat arbitrary since we don't explicitly `deinit` any instances.